cypress/features/regression/actionPopover.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Action Popover component
+  I want to change Alert component properties
+
+  Background: Open Action Popover component page
+    Given I open "Action Popover" component page
+      And I open Action Popover component
```

Comment:
maybe replace string 'component' to 'element'. It would be more readable

Reply:
fixed